src/main/kotlin/com/sourcegraph/cody/config/ui/UpdateChannel.kt

```diff
@@ -4,12 +4,12 @@ import com.intellij.util.ui.PresentableEnum
 
 enum class UpdateChannel(val channelUrl: String?) : PresentableEnum {
   Stable(null as String?),
-  Alpha("https://plugins.jetbrains.com/plugins/alpha/9682");
+  Nightly("https://plugins.jetbrains.com/plugins/alpha/9682");
```

Comment:
I think the link can be update now as well 🆙

Reply:
Done!